### PR TITLE
fix(growth): Fixes crash free rates and Relay warning in Sandbox

### PR DESCRIPTION
**Pull request by roggenkemper**
This PR fixes a few of the issues with the latest update to the Sandbox. First, it lowers the number of users in each aggregate session so there is no longer a warning from Relay. Second, it reduces the number of individual sessions that are sent which should help with the time it takes to create a demo org. Finally, it makes the crash free rate better reflect the thresholds that are set. This PR also allows for the quick demo generation parameters to be an override instead of a copy of the "normal" generation parameters. 

GRW-176 and GRW-175

### CODE REVIEW

File: src/sentry/demo/data_population.py [hunk 1]
```diff
@@ -1231,6 +1234,7 @@ def populate_sessions(self, project, distribution_fn_num: int, mobile: bool, err
         seen_versions = []
         num_versions = 0
         weights = []
+        ind_session_threshold = 0.5
```
Comment (scefali) [inline src/sentry/demo/data_population.py:1237]: @roggenkemper can we make this a setting on `DEMO_DATA_GEN_PARAMS`?

Comment (roggenkemper) [reply src/sentry/demo/data_population.py]: yup!

File: src/sentry/demo/data_population.py [hunk 2]
```diff
@@ -1252,37 +1256,39 @@ def populate_sessions(self, project, distribution_fn_num: int, mobile: bool, err
...
+            if outcome > ind_session_threshold:
+                # send sessions for duration info
+                session_data = {"init": True}
+                self.send_session(
+                    sid, transaction_user["id"], dsn, timestamp, version, **session_data
+                )
+                release_num = int(version.split(".")[-1])
+                threshold = rate_by_release_num[release_num]
+                outcome = random.random()
+                if outcome > threshold:
```
Comment (scefali) [inline src/sentry/demo/data_population.py:1269]: @roggenkemper i'm a bit confused why we are checking the `outcome` twice, seems odd

Comment (roggenkemper) [reply src/sentry/demo/data_population.py:1269]: the first time is to see if we are going to send a session or not, the second time is to determine if the session crashes or not.

Comment (scefali) [reply src/sentry/demo/data_population.py:1269]: @roggenkemper let's just use a different distribution function then I think

Comment (scefali) [reply src/sentry/demo/data_population.py:1269]: on second thought, that might be a bigger refactor. I guess what you have is OK if not ideal.

File: src/sentry/demo/data_population.py [hunk 3]
```diff
@@ -1252,37 +1256,39 @@ def populate_sessions(self, project, distribution_fn_num: int, mobile: bool, err
...
-                    self.safe_send_event(local_event)
-
-                data = {
-                    "status": "crashed",
-                }
-            else:
-                data = {
-                    "status": "exited",
-                }
+            if outcome > ind_session_threshold:
```
Comment (scefali) [inline src/sentry/demo/data_population.py:1260]: @roggenkemper Nit: might be cleaner to do:
```
if outcome < ind_session_threshold:
   continue
```

Comment (roggenkemper) [reply src/sentry/demo/data_population.py:1260]: agree!

File: src/sentry/demo/data_population.py [hunk 4]
```diff
@@ -554,7 +554,13 @@ def get_config(self):
             return settings.DEMO_DATA_GEN_PARAMS
 
     def get_config_var(self, name):
```
Comment (scefali) [inline src/sentry/demo/data_population.py:556]: @roggenkemper Might be worth adding some unit tests for this function

File: tests/sentry/demo/test_demo_org_manager.py [hunk 5]
```diff
@@ -208,3 +212,37 @@ def test_create_mobile_demo_org(
         assert len(Project.objects.filter(organization=org)) == 5
         mock_handle_scenario.assert_called_once_with(mock.ANY, mock.ANY)
         mock_handle_mobile_scenario.assert_called_once_with(mock.ANY, mock.ANY, mock.ANY)
+
+    @override_settings(DEMO_DATA_QUICK_GEN_PARAMS=DEMO_DATA_QUICK_GEN_PARAMS)
+    @mock.patch.object(DataPopulation, "handle_react_python_scenario")
+    @mock.patch.object(DataPopulation, "handle_mobile_scenario")
+    @mock.patch("sentry.demo.demo_org_manager.generate_random_name", return_value=org_name)
+    @mock.patch.object(DataPopulation, "generate_releases")
+    def test_get_config_var(
```
Comment (scefali) [inline tests/sentry/demo/test_demo_org_manager.py:221]: @roggenkemper I think this belongs in a `test_data_population.py` file where we can omit a lot of the setup that's needed for a demo org. The org can really just be a normal org too: `DataPopulation` doesn't care what kind of org it is.

Comment (roggenkemper) [reply tests/sentry/demo/test_demo_org_manager.py:221]: makes sense!

File: tests/sentry/demo/test_data_population.py [hunk 6]
```diff
@@ -0,0 +1,37 @@
...
+@override_settings(
+    DEMO_MODE=True,
+    DEMO_DATA_GEN_PARAMS=DEMO_DATA_GEN_PARAMS,
+    DEMO_DATA_QUICK_GEN_PARAMS=DEMO_DATA_QUICK_GEN_PARAMS,
+)
+class DataPopulationTest(TestCase):
+    def setUp(self):
+        super().setUp()
+        self.react_project = self.create_project(organization=self.organization, platform="react")
+        self.python_project = self.create_project(organization=self.organization, platform="python")
```
Comment (scefali) [inline tests/sentry/demo/test_data_population.py:25]: @roggenkemper why do we need to create the projects? Seems like it's unused.

Comment (roggenkemper) [reply tests/sentry/demo/test_data_population.py:25]: remnant from past from before I moved a lot of the tests over to demo org manager. not needed here anymore.

File: tests/sentry/demo/test_data_population.py [hunk 7]
```diff
@@ -0,0 +1,34 @@
...
+
+
+@override_settings(
+    DEMO_MODE=True,
+    DEMO_DATA_GEN_PARAMS=DEMO_DATA_GEN_PARAMS,
+    DEMO_DATA_QUICK_GEN_PARAMS=DEMO_DATA_QUICK_GEN_PARAMS,
+)
+class DataPopulationTest(TestCase):
+    def setUp(self):
+        super().setUp()
```
Comment (scefali) [inline tests/sentry/demo/test_data_population.py:23]: @roggenkemper You don't need to define this at all FYI

**Review by scefali — APPROVED** [review]

**Pull request merged**